docs/apm-infra-correlation.md

```diff
@@ -0,0 +1,295 @@
+# How to Enable APM-Infra Correlation 
+
+In the APM Service Dashboards, there are charts that indicate the health of
+the underlying infrastructure. The default configuration of Splunk OpenTelemetry 
+Connector automatically configures this for you, but when specifying a custom 
+configuration it is important to keep the following in mind.
+
+
+
+## Configuration to enable APM-Infra Correlation from a standalone VM (Agent)
```

Comment:
I would change "to get your infrastructure data to show up," but I'm not quite sure how to keep the technical accuracy. For example, the user has to do something (enable those things in the config) to get the data to show up, which is different from saying something like "to view your infrastructure data". 

Also, I would change the list to a table.

Reply:
Thats fair, I'll change this to "to view your infrastructure data". 

My only hesitancy with changing to a table is that one column will be extremely long compared to the rest. List is certainly cleaner. Thoughts?